### ISSUE TITLE
it does not work

### ISSUE THREAD

**Issue by xiufengliu** [qunash/chatgpt-advanced#20]
I have tried for many times, without success. Could you help to investiage?

**Comment by herman925** [comment]
It doesn't work at times. I tried turning it on and make a prompt and usually it will generate the results and prompt ChatGPT with a preset, right? But it doesn't and ChatGPT just answers my original question.

**Comment by WindyITS** [comment]
I have the same problem. The extension doesnt work for me. I enabled it but doesnt help

**Comment by daryll-swer** [comment]
@qunash it seems your code broke and is no longer working on the site as of today. Likely they made some changes.

**Comment by qunash** [comment]
Hi all, this is really weird because it still works for me...
Can you provide console logs if thre are any errors?

**Comment by WindyITS** [comment]
Here there is the screen of mi console: 
![image](https://user-images.githubusercontent.com/120594832/207680905-2c4cab1e-69b8-48b0-a4ba-ac66844e3df6.png)

**Comment by qunash** [comment]
Please, check if it works now

**Comment by WindyITS** [comment]
No, still not working. Do you want me to upload a new screenshot?

**Comment by tejush1998** [comment]
I keep getting the error, ChatGPT cant browse the internet...? Something broke?

**Comment by Carnagal** [comment]
Same problem. It worked one time, but never after. No idea why

**Comment by qunash** [comment]
@Carnagal please provide more details/screenshot

**Comment by Carnagal** [comment]
Hello, well your question surprised me so I tested with a more simple question and it worked. I think my request is just too hard for it, It needs to go deeper in web. Sorry for my mistake and may be later with further improvements, it will work.

**Comment by qunash** [comment]
I'm curious to know which prompts it can't handle. Could you list them here or on the [Discord server](https://discord.gg/pHKRutST)?

**Comment by Carnagal** [comment]
> I'm curious to know which prompts it can't handle. Could you list them here or on the [Discord server](https://discord.gg/pHKRutST)?

I have written it in french to make it the most precise I could. But it is a big fail :)

Je vais écrire une liste des #tag de joueurs clash royal ainsi que leurs pseudonymes. Il est possible de savoir dans quel clan chacuns se trouvent avec royal.api  Comme Carnagal https://royaleapi.com/player/Q8LC888Y qui se trouve dans le clan Band of adultes.
Donne moi la liste des pseudos suivi du clan dans lesquels ils se trouvent.
#9G9VVGCR	Magma
#QYP9VGRJR	Maxybestof
#Q9VQLY8	Dofla
#8URYRGPJ	Bembie

**Comment by seriousAA** [comment]
> I'm curious to know which prompts it can't handle. Could you list them here or on the [Discord server](https://discord.gg/pHKRutST)?

I have same problem, it doesn't work.
![image](https://user-images.githubusercontent.com/48859437/231220549-a6af3760-e5e6-44c3-97ba-26372e6853c0.png)

**Comment by seriousAA** [comment]
And my console logs
![image](https://user-images.githubusercontent.com/48859437/231220855-4559c92e-dabc-4cba-aebb-dc78644e039d.png)
